server/server.js

```diff
@@ -103,7 +103,7 @@ app.use(loopback.urlNotFound());
  */
 
 // The ultimate error handler.
-app.use(loopback.errorHandler());
+app.use(loopback.strongErrorHandler());
```

Comment:
This is weird, apparently the person upgrading this project added JSON files but forgot to change `server/server.js` to match the new template. Let's revert all changes outside `templates` to keep this PR focused.